lms/envs/aws.py

```diff
@@ -272,6 +272,9 @@
 # Queue to use for updating persistent grades
 RECALCULATE_GRADES_ROUTING_KEY = ENV_TOKENS.get('RECALCULATE_GRADES_ROUTING_KEY', LOW_PRIORITY_QUEUE)
 
+# Queue to use for updating grades due to grading policy change
+POLICY_CHANGE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
This has to be the ENV_TOKENS.get() syntax like RECALCULATE_GRADES_ROUTING_KEY otherwise it ignores lms.env.json